opencti-platform/opencti-graphql/src/database/file-search.js

```diff
@@ -62,7 +62,19 @@ export const elIndexFiles = async (context, user, files) => {
         uploaded_at: file.uploaded_at,
       };
       const documentBody = buildIndexFileBody(internal_id, fileObject, entity);
-      await elIndex(INDEX_FILES, documentBody, { pipeline: 'attachment' });
+      let success = true;
+      await elIndex(INDEX_FILES, documentBody, { pipeline: 'attachment' }).catch((err) => {
+        // catch & log error
+        logApp.error(err);
+        success = false;
+      });
+      if (!success) {
```

Comment:
I tried but I had an error because it's async, I'm not sure it would work if I call just `elIndex(INDEX_FILES, documentWithoutFileData);` without await

Reply:
Oh yes, i think in this case its more usual to use a catch block instead of the promise catch

Reply:
it works inside the promise catch